drivers/gles3/shaders/skeleton.glsl

```diff
@@ -164,10 +174,13 @@ void main() {
 	m += GET_BONE_MATRIX(bones.z, bones_a.z, in_weight_attrib.z);
 	m += GET_BONE_MATRIX(bones.w, bones_a.w, in_weight_attrib.w);
 
+	mat4 skeleton_matrix = mat4(vec4(skeleton_transform_x, 0.0, 0.0), vec4(skeleton_transform_y, 0.0, 0.0), vec4(0.0, 0.0, 1.0, 0.0), vec4(skeleton_transform_offset, 0.0, 1.0));
```

Comment:
This is only for 2D right?

Reply:
Yep, that code is only included when calculating 2D skeletons